tests/nn/pipe_process/test_pipe.py

```diff
@@ -31,9 +31,11 @@
     get_pipeline_parallel_group,
     initialize_model_parallel,
 )
-from fairscale.nn.pipe import LazyModule, Pipe
+from fairscale.nn.pipe import LazyModule, MultiProcessPipe
 from fairscale.utils.testing import get_worker_map, set_random_seed, torch_spawn, torch_version
 
+Pipe = MultiProcessPipe
```

Comment:
Quick question: Would be it confusing if we just import MultiProcessPipe as Pipe ( i mean `from fairscale.nn.pipe import MultiProcessPipe as Pipe`)

I see you have are using in couple of tests.

Not sure if it will be good for readability, but just putting it out there :)